lib/rules/indent.js

```diff
@@ -61,6 +66,24 @@ module.exports = function(context) {
         }
     }
 
+    if (typeof options.VariableDeclarator === "number") {
+        options.VariableDeclarator = {
+            var: options.VariableDeclarator,
+            let: options.VariableDeclarator,
+            const: options.VariableDeclarator
+        };
+    } else {
```

Comment:
in else condition just use object-assign dependency.
Its already being used inside this module as `assign`.